example/multiTenancyExampleOnchainPrivacyEventLogs/subscribeWebSockets.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const path = require("path");
+const Web3 = require("web3");
+const EEAClient = require("../../src");
+
+const { besu, orion } = require("../keys");
+const { createHttpProvider } = require("../helpers.js");
+
+// use an orion key that is not a member of the group (eg orion.node11.jwt)
+// to demonstrate that they can't create a subscription
+const node = new EEAClient(
+  new Web3(createHttpProvider(orion.node1.jwt, besu.node1.url)),
+  2018
+);
+const params = JSON.parse(fs.readFileSync(path.join(__dirname, "params.json")));
+
+async function run() {
+  const { privacyGroupId, contractAddress: address, blockNumber } = params;
+
+  const filter = {
+    address,
+    fromBlock: blockNumber
+  };
+
+  console.log("Installing filter", filter);
+
+  // Create subscription
+  return node.priv
+    .subscribe(privacyGroupId, filter, (error, result) => {
+      if (!error) {
+        console.log("Installed filter", result);
+      } else {
+        console.error("Problem installing filter:", error);
+        throw error;
+      }
+    })
+    .then(async subscription => {
+      // Add handlers for incoming events
+      subscription
+        .on("data", log => {
+          if (log.result != null) {
+            // Logs from subscription are nested in `result` key
+            console.log("LOG =>", log.result);
+          } else {
+            console.log("LOG =>", log);
+          }
+        })
+        .on("error", console.error);
+
+      // Unsubscribe and disconnect on interrupt
+      process.on("SIGINT", async () => {
+        console.log("unsubscribing");
+        await subscription.unsubscribe((error, success) => {
+          if (!error) {
+            console.log("Unsubscribed:", success);
```

Comment:
NIT: Should this include the filter ID that we have subscribed?

Reply:
I don't think the filterID is actually exposed here atm so not making this change in this PR